app/routes/image-sets/new.js

```diff
@@ -1,12 +1,19 @@
+import Ember from 'ember';
 import ImageSetRoute from '../image-set';
+import {defaultLocation} from 'lion-guardians/utils/units';
 
 export default ImageSetRoute.extend({
   controllerName: 'image-set',
+  defaultLocation: defaultLocation,
 
   model: function() {
-    var currentUser = this.get('toriiSession.currentUser');
+    var currentUser = this.get('toriiSession.currentUser'),
+        defaultLocation = this.get('defaultLocation');
+
     return this.store.createRecord('image-set', {
-      user: currentUser
+      user: currentUser,
+      latitude: Ember.get(defaultLocation, 'lat'),
```

Comment:
actually, now that I think of it, it may be most appropriate to make this a default value in the model definition for an image set. In image-set.js:
```
...
latitude: DS.attr('number', {defaultValue: defaultLocation.latitude});
```

Also, fwiw, I would prefer to use the same property name everywhere (`latitude` instead of `lat`), for consistency. It's a little unexpected that the default latitude value for the image set is a property called 'lat' instead of 'latitude'.
Longitude is more error-prone because some people abbreviate it 'long' and others 'lng'.